src/js/views/patients/sidebar/flow/flow-sidebar_views.js

```diff
@@ -76,6 +76,7 @@ const LayoutView = View.extend({
   modelEvents: {
     'change:_state': 'showOwner',
     'change:_owner': 'showFlow',
+    'change:updated_at': 'showTimestamps',
```

Comment:
This doesn't do anything. But the `updated` timestamp in the flow sidebar is updating correctly on model change. 

So I'm not sure what's going on here yet.

Reply:
I'm noticing other weird things with the flow sidebar, in terms of updating on a web socket model change.

Name and owner update correctly. But details and state do not.

I'm going to investigate/address those in this story [here](https://app.shortcut.com/roundingwell/story/56894/update-flow-sidebar-on-web-socket-model-changes).

Reply:
For the first comment above, the `change:updated_at` model event is indeed working. I just had the test setup incorrectly. The flow sidebar needs to be open before the ws notification comes in. I updated the test to do that.